src/main/java/io/gingersnapproject/cdc/event/BackendStartedEvent.java

```diff
@@ -0,0 +1,5 @@
+package io.gingersnapproject.cdc.event;
+
+import java.net.URI;
+
+public record BackendStartedEvent(URI uri) { }
```

Comment:
Instead of creating a file per event, is it cleaner to create an Events class that hosts all records?

Something like:

```java
public class Events {
   public record BackendFailed(URI uri, Throwable throwable) { }

   public record BackendStarted(URI uri) { }
   ...
}
```

We can then just reference them via `Events.BackendStarted` etc.